### DISCUSSION TITLE
Pengumuman

### DISCUSSION THREAD

**Discussion by misskecupbung** [awsugid/forum#1] [Announcements]
Selamat datang di forum AWS UG Indonesia!

**Comment by rioastamal** [comment]
Halo :D !!

Saya rasa ini bakalan berguna untuk arsip dan media belajar karena kadang diskusi di Telegram gampang tenggelam padahal banyak informasi bermanfaat disana. Ditambah saya bisa pakai code formatting disini. Asyeek

```javascript
console.log("Hello World")
```

**Comment by za** [comment]
Halo juga 😎
